scripts/python/clear_port_macs.py

```diff
@@ -37,4 +39,14 @@ def main():
 
 if __name__ == '__main__':
     logger.create()
-    main()
+
+    if len(sys.argv) > 2:
+        try:
+            raise Exception()
+        except Exception:
+            sys.exit('Invalid argument count')
```

Comment:
Why raise an exception here? Wouldn't this accomplish the same:

```python
if len(sys.argv) > 2:
    sys.exit('Invalid argument count')
```

This pattern is repeated quite a few times throughout the commit - I'll skip the "ditto" comments.

Reply:
Yep.  I copied a block of code and then thinned it some without giving it enough thought.